src/DetectorConstruction.h

```diff
@@ -19,9 +19,13 @@ class DetectorConstruction: public G4VUserDetectorConstruction
 public:
   DetectorConstruction();
   virtual ~DetectorConstruction();
+  bool GetDetectorConfiguration() const { return detectorConfiguration;}
```

Comment:
I assume this method is no longer needed if we use the ConfigManager.

Reply:
It looks like this is slightly different than what the ConfigManager is doing.. It looks like `detectorConfiguration` within `DetectorConstruction.h` is what is used to select between VD and HD detector  constructions. It also appears that there's no way to set this to false, and thus allowing for a VD construction.

Reply:
Included a commit which modifies the constructor to allow selection between VD and HD.